src/main/java/net/snowflake/client/jdbc/SnowflakeFileTransferAgent.java

```diff
@@ -1934,8 +1935,9 @@ static Set<String> expandFileNames(String[] filePathList, String queryId)
 
     // For each location, list files and match against the patterns
     for (Map.Entry<String, List<String>> entry : locationToFilePatterns.entrySet()) {
+      java.io.File dir = null;
```

Comment:
File class can be used without the package (here and in other places below) - it's already imported